pkgs/top-level/all-packages.nix

```diff
@@ -19532,6 +19532,8 @@ with pkgs;
 
   lurk = callPackage ../development/tools/lurk { };
 
+  maizzle-cli = callPackage ../development/tools/maizzle-cli { };
```

Comment:
Should this be
```suggestion
  maizzle = callPackage ../development/tools/maizzle { };
```
?

Reply:
The npm package is called @maizzle/cli but the binary is called maizzle. I dont think there will be misunderstanding, calling it maizzle is fine IMO.